src/Services/Signatures/HashSignature.php

```diff
@@ -6,17 +6,17 @@
 
 class HashSignature implements SignatureInterface
 {
-    const ALGORITHM = 'sha256';
+    public const ALGORITHM = 'sha256';
```

Comment:
Are these really public? I understand that the default of these is public, but I think these should only be used in this class, so should be private